src/include/clSPARSE.h

```diff
@@ -464,6 +464,15 @@ CLSPARSE_EXPORT clsparseStatus
 clsparseDdense2csr(const cldenseMatrix* A, clsparseCsrMatrix* csr,
                    const clsparseControl control);
 
+//SpM x SpM
```

Comment:
Make sure to decorate this function with the appropriate doxygen comments now.  From now on, when we introduce new functions into clsparse.h, they should come with the appropriate documentation.